### DISCUSSION TITLE
關於Arcahv的梯子

### DISCUSSION THREAD

**Discussion by WhiteCloud0106** [Lost-MSth/Arcaea-server#185] [Q&A] [answered]
如果要跑arcahv異象是跟一般世界曲子的梯子一樣就可以了麼？還是要加什麼其他東西
我只知道前置異象解鎖要設定成暴風異象，不太懂其他有沒有要額外設定
不知道怎麼抓官方梯子，能順便教這個更好了（（（

**Comment by Lost-MSth** [comment, marked as answer]
似乎没有特殊要求，就是首次游玩时触发异象 <https://arcwiki.mcd.blue/Arcahv>

**Reply by WhiteCloud0106** [reply]
順便問一下Beyond Chain是可以透過甚麼方式弄出來嗎
還是只有原版對應ID才能組成

**Reply by Lost-MSth** [reply]
写个 `chain_info`  #148

**Reply by WhiteCloud0106** [reply]
大致了解了，不過想問一下關於"id": "test"那邊，test的部分是只要五張地圖都使用相同的id就會自動組合嗎，還是需要針對test去寫一個什麼文件呢？
再來就是pragmatism resurrection跟vicious anti heroism兩隻不同的順序是各怎麼寫

**Reply by Lost-MSth** [reply]
是的，只需要一个相同的 chain_id 就会认为是一条链上的
没懂，什么顺序

**Reply by WhiteCloud0106** [reply]
就是pragmatism的那個byd chain是五張譜面一張一張推進的
但vicious heroism的則是從兩邊推進的
不知道怎麼寫才能寫出這兩種差別？

**Reply by Lost-MSth** [reply]
emmm 如果我没猜错的话，"beyond_chain_2" 是一个硬编码的 "chain_id"，其 0~4 的位置是特殊的

**Comment by WhiteCloud0106** [comment]
![image](https://github.com/user-attachments/assets/e1b03b1e-8e90-4b8f-9244-52095e0f777c)
不好意思，想問一下除了這個還需要添加什麼條件嗎
因為單純這樣設定好像會直接可以解鎖了(亦或是我的殼子哪邊有問題?

**Reply by Lost-MSth** [reply]
我也不清楚，要不把上面留空的全都删掉？有可能 "tempestissimo_challenge" 已经通过了，或者你说的壳子有问题也有可能。
